clusterloader2/api/types.go

```diff
@@ -142,15 +142,28 @@ type TuningSet struct {
 	GlobalQPSLoad *GlobalQPSLoad `json: globalQPSLoad`
 }
 
+// MeasurementInstanceConfig is a structure that contains the Instance for wrapper measurements along with optional params.
+type MeasurementInstanceConfig struct {
+	// Identifier is an interface{} which could be a string or map of {name: value} that differentiates measurement instances of the same method.
```

Comment:
Comment is no longer valid - please unify it with the comment for Identifier in Measurement struct.